vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1913,14 +1902,6 @@ int vtkF3DRenderer::UpdateLights()
   return lightCount;
 }
 
-//----------------------------------------------------------------------------
-bool vtkF3DRenderer::IsBackgroundDark()
```

Comment:
why is it removed?

Reply:
because it is not used at all, see: https://github.com/f3d-app/f3d/issues/2337